openedx/core/djangoapps/programs/utils.py

```diff
@@ -113,6 +114,16 @@ def get_programs_for_credentials(user, programs_credentials):
     return certificate_programs
 
 
+def get_display_category(program):
+    display_candidate = None
+    if program and program.get('category', None):
+        if program.get('category', None) == 'xseries':
+            display_candidate = 'XSeries'
+        else:
+            display_candidate = program.get('category', '').capitalize()
+    return display_candidate
```

Comment:
should we avoid returning `None` from this?   empty string, at least, seems safer.